diff --git a/node_modules/@cloudflare/vite-plugin/dist/index.mjs b/node_modules/@cloudflare/vite-plugin/dist/index.mjs
index b150002..3e9ca81 100644
--- a/node_modules/@cloudflare/vite-plugin/dist/index.mjs
+++ b/node_modules/@cloudflare/vite-plugin/dist/index.mjs
@@ -26569,7 +26569,7 @@ const outputConfigPlugin = createPlugin("output-config", (ctx) => {
 			if (ctx.resolvedPluginConfig.type === "workers") {
 				const inputConfig = ctx.getWorkerConfig(this.environment.name);
 				if (!inputConfig) return;
-				const entryChunk = Object.values(bundle).find((chunk) => chunk.type === "chunk" && chunk.isEntry && chunk.name === MAIN_ENTRY_NAME);
+				const entryChunk = Object.values(bundle).find((chunk) => chunk.type === "chunk" && chunk.isEntry);
 				assert(entryChunk, `Expected entry chunk with name "${MAIN_ENTRY_NAME}"`);
 				const isEntryWorker = this.environment.name === ctx.resolvedPluginConfig.entryWorkerEnvironmentName;
 				outputConfig = {
