diff --git a/dist/config/index.cjs b/dist/config/index.cjs
index 9357cce3f218c6051c62d2ec7a0d56d59eb57743..d8e0db1a33a27645bc941b2d3703fb0cd2ec628e 100644
--- a/dist/config/index.cjs
+++ b/dist/config/index.cjs
@@ -181,7 +181,7 @@ function createConfigPlugin() {
         ...workerdBuiltinModules,
         ...import_node_module.builtinModules.concat(import_node_module.builtinModules.map((m) => `node:${m}`))
       ]);
-      config.test.pool ??= "@cloudflare/vitest-pool-workers";
+      config.test.pool ??= import_node_path2.default.resolve(__dirname, "../pool/index.mjs");
     },
     resolveId(id) {
       if (id === "cloudflare:test") {
